rpm/dovecot-ceph-plugin.spec

```diff
@@ -18,19 +43,21 @@ Release:	0%{?dist}
 URL:		https://github.com/ceph-dovecot/dovecot-ceph-plugin
 Group:		Productivity/Networking/Email/Servers
 License:	LGPL-2.1
-Source:		%{name}_%{version}-%{release}.tar.gz
+Source:		%{name}-%{version}.tar.xz
```

Comment:
This would break the Deutsche Telekom build we are using. Is it possible to overwrite "Source" from the outside?

Reply:
"from the outside" means?

Reply:
The point is that make_dist.sh builds by default a %{name}-%{version}.tar.xz

Reply:
For the DTAG build we have to modify several path names to adhere to their internal standards. These path names are overridden by the script that is executing the build process. To avoid collision with potential OS packages we use %{release}, which a standard field, to mark a custom build. If you don't mind, I would suggest changes to make_dist.sh to support a release or dist, to not break our established build process